src/bson_mapper/CMakeLists.txt

```diff
@@ -119,6 +119,13 @@ install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
         REGEX ".+/test" EXCLUDE
 )
 
+install(DIRECTORY ${CMAKE_SOURCE_DIR}/src/third_party/cereal/include/cereal/
+    DESTINATION "include/cereal"
```

Comment:
The project makes just fine and tests run successfully with this, but when I run the examples they can't find `cereal/cereal.hpp` when included by `bson_mapper/bson_archiver.hpp`